site/src/components/checkbox.md

```diff
@@ -58,14 +58,40 @@ Here is an example of a group of checkboxes. Default spacing is added between ea
 </label>
 ```
 
-### Size
+### Sizes
 
-`.checkbox-sm` is used with `.checkbox` to display a smaller checkbox.
+`.checkbox-sm`/`.checkbox-lg` used in conjunction with `.checkbox` makes checkbox a little bigger or smaller than the default.
 
```

Comment:
`.checkbox-sm`/`.checkbox-lg` can be used to make a checkbox bigger or smaller than the default.

Reply:
Updated